api.yaml

```diff
@@ -754,7 +814,7 @@ components:
     AccountOwner:
       type: object
       properties:
-        acountId:
+        accountId:
           type: string
```

Comment:
This is a typo fix. The existing endpoints works as expected with the typo.

Reply:
Would this not change the api contract? If the current `/v1/account/owner` endpoint expects to have an object submitted with an "acountId", will changing this force users of that endpoint to change the object being POSTed?

Reply:
The json declaration of "accountId" is here: https://github.com/asecurityteam/asset-inventory-api/blob/master/pkg/handlers/v1/insert_account_owner.go#L12. The endpoint should work with "accountId".

Reply:
"acountId" does not match "accountId" in https://github.com/asecurityteam/asset-inventory-api/blob/master/pkg/handlers/v1/insert_account_owner.go#L12.

Reply:
If AccountOwner is not in use yet this also makes sense to me.

Reply:
The /v1/account/owner endpoint works with and without the typo fix, but I decided to fix it to match the struct in insert_account_owner.go.

Reply:
Ooops, thank you! And I'm sorry about the follow up comment, I thought I had refreshed the page to see if you had replied but I guess I hadn't!

Reply:
Thank you for asking so I can clarify